### ISSUE TITLE
Allow setting individual border's properties

### ISSUE THREAD

**Issue by keogami** [dunst-project/dunst#1136]
Hello, i am trying to rice dunst's notification boders and would like to customize the properties of a particular edge.
In particular, i am trying to achieve the following look
![image](https://user-images.githubusercontent.com/41939011/210462940-b62eedef-d921-428f-90ca-8abdada32b8e.png)

Reading the docs, i get the impression that this is not currently possible. But I believe it will be a good feature to allow customizing the:
1. Width of individual borders
2. Color of individual borders
3. And perhaps, the dash style for individual borders

**Comment by fwsmit** [comment]
Instead of adding a new setting key you could also change `frame_width` to allow a list of numbers.
E.g.
```
frame_width= (1,2,3,4) # set all borders individually
frame_width=1 # set all borders at once
```

It would also be good to look at mako's configuration to see how they do it. I believe they do something similar.

**Comment by keogami** [comment]
Hmm that sounds like a good solution but dunst allows overriding settings based on urgency and rules (if i understand this correctly). So, consider the case where all the borders have been set to have some colors like
```
# (I'm assuming the order as: top right bottom left)
frame_color = ("red", "red", "blue", "blue") # for example
```
now, if for certain urgency and rules, someone wants to adjust the bottom and left borders they will have to restate top and right for no reason
```
frame_color  = ("red", "red", "black", "black")
```

And they will have to do that for every special rule they make and urgency. Which is a bad experience and a lot of copy/pasting for no reason.

Having separate settings for each border won't have theses issues.

**Comment by fwsmit** [comment]
That's true. You could also allow empty values for keeping the old color:

```
frame_color = ( , , , "red")
```

But yeah, it could be discussed more when someone wants  to implement it :)

**Comment by bynect** [comment]
I am planning an overhaul of dunst frame drawing
